test/dataframe.jl

```diff
@@ -347,7 +396,7 @@ module TestDataFrame
         @test udf == DataFrame(Any[Union{Int, Missing}[1, 2], Union{Int, Missing}[1, 5],
                                    Union{Int, Missing}[2, 6], Union{Int, Missing}[3, 7],
                                    Union{Int, Missing}[4, 8]], [:id, :a, :b, :c, :d])
-        @test all(isa.(udf.columns, Vector{Union{Int, Missing}}))
+        @test all(isa.(udf.columns[2:end], Vector{Union{Int, Missing}}))
```

Comment:
Also test the type of the first column. Same below.

Reply:
fixed